controller/utils.go

```diff
@@ -255,6 +255,8 @@ func replacePodWorker(
 			}
 
 			logger.Infof("pods remaining to replace: %d", len(pods))
+
+			return nil
```

Comment:
is the for loop necessary if you're returning nil here?

Reply:
Sorry, forgot the if.